bootstrap-sharpmake.bat

```diff
@@ -6,12 +6,19 @@ pushd "%~dp0"
 
 set SHARPMAKE_EXECUTABLE=bin\debug\Sharpmake.Application.exe
 
-if not defined VS140COMNTOOLS (
-    echo ERROR: Cannot determine the location of the VS Common Tools folder.
-    goto error
+if defined VS140COMNTOOLS (
+    call "%VS140COMNTOOLS%VsMSBuildCmd.bat"
+    goto buildcmdcalled
 )
+if defined VS150COMNTOOLS (
+    call "%VS150COMNTOOLS%VsMSBuildCmd.bat"
+    goto buildcmdcalled
+)
+
+echo ERROR: Cannot determine the location of the VS Common Tools folder.
+goto error
 
-call "%VS140COMNTOOLS%VsMSBuildCmd.bat"
+:buildcmdcalled
```

Comment:
Revert changes to that file, with latest version of sharpmake they shouldn't be needed

Reply:
Oh, sorry, a different change got mixed in with this, I'll get this cleaned up.